pkg/vm/engine/disttae/txn.go

```diff
@@ -212,8 +215,16 @@ func (txn *Transaction) IncStatementId() {
 
 // Write used to write data to the transaction buffer
 // insert/delete/update all use this api
-func (txn *Transaction) WriteBatch(typ int, databaseId, tableId uint64,
-	databaseName, tableName string, bat *batch.Batch, dnStore DNStore) error {
+func (txn *Transaction) WriteBatch(
```

Comment:
this function, added param primaryIdx, and -1 all need to be documented.

Reply:
added